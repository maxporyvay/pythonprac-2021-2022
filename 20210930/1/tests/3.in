124,6346,8568,14,463,547
