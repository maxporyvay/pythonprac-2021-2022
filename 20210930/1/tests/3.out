[547, 6346, 8568, 463, 124, 14]
